src/app/course/add-course/add-course.component.ts

```diff
@@ -1,13 +1,15 @@
-import { Component, OnInit } from "@angular/core";
+import { Component, OnInit, ChangeDetectionStrategy } from "@angular/core";
 import { FormGroup, FormControl, Validators } from "@angular/forms";
 import { Router } from "@angular/router";
 import { Course } from "../_models/course";
```

Comment:
please, devide imports groups by empty line.
Imports can by devided into 3 groups:
external, own (your domen) libs and internal imports